modules/nf-core/bcftools/reheader/main.nf

```diff
@@ -21,8 +21,9 @@ process BCFTOOLS_REHEADER {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def update_sequences = fai ? "-f $fai" : ""
-    def new_header       = header ? "-h $header" : ""
+    def update_sequences = fai ? "--fai $fai" : ""
+    def new_header       = header ? "--header $header" : ""
+    def rename_samples   = samples ? "--samples $samples" : ""
```

Comment:
good idea :p

Reply:
or maybe `..._argument` is better?

Reply:
I suggested command, because it's what I used already in such cases:
https://github.com/nf-core/modules/blob/master/modules/nf-core/gatk4/applybqsr/main.nf#L27

Reply:
but _argument looks ok to me.
We need to come up with a best practice and stick to it in every module

Reply:
I see! But they aren't really commands though? :grin: